### PR TITLE
ITO-230: Add NPM publish github action

### PR DESCRIPTION
**Pull request by dmccrae-ubif**

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: NPM Publish
+on:
+  push:
+    branches:
+      - ITO-230
```
Comment (dmccrae-ubif) [inline .github/workflows/npm-publish.yml:5]:
```suggestion
      - master
```

**Review by dave9011 — APPROVED** [review]

**Review by dave9011 — CHANGES_REQUESTED** [review]
Let's update the README file with updated installation instructions

**Review by dave9011 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -11,13 +11,13 @@ A complete example can be found at [http://jonashartmann.github.io/webcam-direct
 ## Download
 [![NPM](https://nodei.co/npm/webcam.png?compact=true)](https://nodei.co/npm/webcam/)
 
-#### Using [NPM](http://www.npmjs.com)
+#### Using the command line
 ```shell
-npm install webcam
+npm install @ubreakifix/webcam@3.2.13
```
[the remark this answers is not in the document]
Comment (dmccrae-ubif) [reply README.md:16]: can exclude the version as it then goes off package.json

Comment (dmccrae-ubif) [inline README.md:16]:
```suggestion
npm install @ubreakifix/webcam
```

**Pull request merged**